### ISSUE TITLE
Add Firebase dev support

### ISSUE THREAD

**Issue by ultraon** [GouravShDev/codersgym#18]
I wanted to contribute by adding iOS support. But I don't have configuration for Firebase. I can try to extract it from apk (reverse engineering), but maybe there is another approach.

Can you help, please?

**Comment by GouravShDev** [comment]
hi @ultraon, 
Thank you for your proposal to contribute! The Firebase configuration files are ignored in the Git repository, so you won’t need them to proceed. However, you can definitely contribute by adding support for iOS. I currently don’t have access to a Mac to set up iOS configurations myself, so your help in creating the iOS setup files would be greatly appreciated.

Just a heads-up, I’m not planning to upload the app to the App Store for now, but having iOS support ready will be a great addition for future plans. Let me know if you need any further details to get started!

**Comment by GouravShDev** [comment]
@ultraon You are right. Contributor won't be able to build app without firebase config files. I'll add it in my next commit. And for iOS I can provide you the google-services.plist file. You can add that along with the ios support.

**Comment by ultraon** [comment]
@GouravShDev thank you, I will proceed then.

**Comment by GouravShDev** [comment]
#27 fixes this
